pkg/frontend/authenticate_test.go

```diff
@@ -10242,307 +10242,6 @@ func TestDoAlterStage(t *testing.T) {
 
 }
 
```

Comment:
TestDoCheckFilePath should have a new implementation instead of removing it.

Reply:
DoCheckFilePath is doing nothing but simply call stage util function.  Not much value to have UT test on a few lines code.